azurerm/internal/services/storage/storage_management_policy_data_source.go

```diff
@@ -53,6 +53,29 @@ func dataSourceStorageManagementPolicy() *schema.Resource {
 										Elem:     &schema.Schema{Type: schema.TypeString},
 										Set:      schema.HashString,
 									},
+
+									"match_blob_index_tag": {
```

Comment:
Is this another "not yet in global property?"
```
------- Stdout: -------
=== RUN   TestAccStorageManagementPolicy_blobIndexMatch
=== PAUSE TestAccStorageManagementPolicy_blobIndexMatch
=== CONT  TestAccStorageManagementPolicy_blobIndexMatch
    testing.go:620: Step 3/6 error: Error running apply: exit status 1
        
        Error: creating Azure Storage Management Policy "/subscriptions/*******/resourceGroups/acctestRG-storage-210426173350412751/providers/Microsoft.Storage/storageAccounts/unlikely23exst2acctu9ru0": storage.ManagementPoliciesClient#CreateOrUpdate: Failure responding to request: StatusCode=400 -- Original Error: autorest/azure: Service returned an error. Status=400 Code="FeatureNotSupportedForAccount" Message="Tag based filtering is not supported for the account."
        
          on terraform_plugin_test.tf line 30, in resource "azurerm_storage_management_policy" "test":
          30: resource "azurerm_storage_management_policy" "test" {
        
--- FAIL: TestAccStorageManagementPolicy_blobIndexMatch (212.08s)
FAIL

------- Stderr: -------
2021/04/26 17:33:49 [DEBUG] not using binary driver name, it's no longer needed
2021/04/26 17:33:50 [DEBUG] not using binary driver name, it's no longer needed
```

Reply:
No. It is released globally. It needs enabled the feature in the subscription. https://azure.microsoft.com/en-us/blog/manage-and-find-data-with-blob-index-for-azure-storage-now-in-preview/. I've added this into the doc, here https://github.com/terraform-providers/terraform-provider-azurerm/pull/11163/files#diff-26c23fcfb7ed0a7939dcabd4959f025c9c94e7fbe5a618c5c13e90462358a09bR108